kratos/utilities/mortar_utilities.h

```diff
@@ -766,95 +755,85 @@ class MortarUtilities
 template<> 
 inline void MortarUtilities::ResetValue<Variable<double>, Historical>(
         ModelPart& rThisModelPart,
-        Variable<double>& ThisVariable, 
-        const bool InvertedPair
-        ) {
+        Variable<double>& ThisVariable
+        ) 
+{
     NodesArrayType& nodes_array = rThisModelPart.Nodes();
     
     // We set to zero
```

Comment:
isn't there a function existing for this in the VariableUtils?
Before you couldn't use it but now I think you can

Reply:
True, but it is possible to do it since recently